### PR TITLE
implemented contact create view

### PR DESCRIPTION
**Pull request by kadimarss**

### CODE REVIEW

File: app/Http/Controllers/ContactController.php [hunk 1]
```diff
@@ -30,6 +29,7 @@ public function index()
         return view('contact.index', compact('contacts'));
     }
 
+
```
Comment (JurgenMik) [inline app/Http/Controllers/ContactController.php]: Is this new line necessary?

File: resources/js/Pages/Contact/Create.js [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+import Checkbox from '@mui/material/Checkbox';
+
+export default function Create(props) {
+    const onHandleChange = (event) => {
+        setData(event.target.name, event.target.type === 'checkbox' ? event.target.checked : event.target.value);
+    };
+
+    const submit = (e) => {
+        e.preventDefault();
+
```
Comment (JurgenMik) [inline resources/js/Pages/Contact/Create.js:16]: Is this new line necessary?

File: resources/js/Pages/Contact/Create.js [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+        <Authenticated
+            auth={props.auth}
+            errors={props.errors}
+            header={<h2 className="font-semibold text-xl text-gray-800 leading-tight">Create contact</h2>}
+        >
+            <ValidationErrors errors={errors} />
+                <Head title="Create" />
+                    <div className="max-w-2xl w-full mx-auto my-8 flex flex-col sm:justify-center items-center pt-6 sm:pt-0 bg-gray-100 ">
+                        <form onSubmit={submit} className="bg-white px-5 py-3">
+                            <div>
```
Comment (JurgenMik) [inline resources/js/Pages/Contact/Create.js:39]: It is suggested to use white space before the element:

```suggestion
                            <div className="mb-4">
```

File: resources/js/Pages/Contact/Create.js [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+                                    name="name"
+                                    value={data.name}
+                                    className="mt-1 block w-full"
+                                    autoComplete="name"
+                                    onChange={onHandleChange}
+                                    required
+                                />
+                            </div>
+
+                            <div className="mt-4">
```
Comment (JurgenMik) [inline resources/js/Pages/Contact/Create.js]: It is suggested to use white space before the element:
```suggestion
                            <div className="mb-4">
```

File: resources/js/Pages/Contact/Create.js [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+                                <formGroup>
+                                    <FormControlLabel control={<Checkbox name="employee" onChange={onHandleChange} />} label="Employee"/>
+                                </formGroup>
+                            </div>
+
+                            <div className="flex items-center justify-center mt-5">
+                                <Button className="ml-4" type="submit" processing={processing}>
+                                    Add
+                                </Button>
```
Comment (JurgenMik) [inline resources/js/Pages/Contact/Create.js:116]: Make the form elements and the button same width?

**Review by JurgenMik — CHANGES_REQUESTED** [review]
Remove required validation from frontend components.
It is suggested to use white space before the element.

File: resources/js/Pages/Contact/Create.js (hunk 2 above)
Comment (kadimarss) [reply resources/js/Pages/Contact/Create.js:16]: Yes.

**Review by JurgenMik — APPROVED** [review]

**Pull request merged**